quiz.sql

```diff
@@ -22,22 +27,35 @@ CREATE DATABASE westeros;
 
 -- your code starts here
 
+INSERT INTO people(name, title, quote, image, age)
+  VALUES ('Cersei Baratheon','King Mother and Lady of Casterly Rock','The gods have no mercy, that’s why they’re gods.', 'http://i.imgur.com/jYeeCJ3.jpg,40')
```

Comment:
You lumped `age`'s value in with `image` here. To the RDBMS, you failed to specify a value for `age`.